src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -647,9 +657,7 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       return null;
     }
     const { applications } = this.props;
-    const newIndexPatternUrl = applications.getUrlForApp('management', {
-      path: 'opensearch-dashboards/indexPatterns',
-    });
+    const newIndexPatternUrl = applications.getUrlForApp('indexPatterns');
```

Comment:
```suggestion
    const newIndexPatternUrl = applications.getUrlForApp('management', {
      path: 'opensearch-dashboards/indexPatterns',
    });
```
Please do not change the index pattern url in this PR.

Reply:
Have modified it according to your suggestion.